surveys/sphinx-survey-2025.yml

```diff
@@ -14,81 +14,70 @@
 # People who interact with documentation, especially those who contribute to
 # or extend documentation. Use of Sphinx is encouraged but not required.
 
-title: "Sphinx Survey 2025"
+title: "Sphinx Documentation Survey (2025)"
 questions:
-- title: "Sphinx Usage Survey 2025"
+- title: "Sphinx Documentation Survey (2025)"
   type: statement
   description: >-
-    Sphinx Documentation User Survey 2025
+    We’re running this survey to understand requirements for documentation
+    tools in 2025, how they are used, and how well tools support their users.
+    This survey is being run on behalf of the [Sphinx] community, but is
+    open to anyone who works with or contributes to documentation, regardless
+    of whether you use Sphinx.
 
-    Help the Sphinx community understand who is using Sphinx, how they are
-    using it, and how well it supports the people using it. We will use this
-    information to aid community discussion about improvements to Sphinx.
+    We will use this information to aid community discussion about improvements
+    to Sphinx. Results will be public, with any identifying information removed.
+    Open text field questions may be quoted, but will not be shown in full.
 
-    Results will be public with identifying information removed. Open text
-    field questions may be quoted, but will not be shown in full.
+    This survey is run by the Sphinx community and [Quansight Labs]; survey
+    data will be deleted from any private accounts once made public. If you
+    have any questions, contact Tania Allard (Director of Quansight Labs)
+    at tallard@quansight.com and mention Sphinx Survey 2025 in the subject line.
 
-    This survey is run by the Sphinx community and Quansight Labs; survey 
-    data will be deleted from any private accounts once made public. If you 
-    have any questions, reach out to tallard@quansight.com and mention 
-    Sphinx Survey 2025 in the subject line.
+    [Sphinx]: https://www.sphinx-doc.org/
+    [Quansight Labs]: https://quansight.com/labs/
 
-- title: "Tell us about you"
-  description: >-
-    First, we'd like to know about how you work and what tools you use.
-  type: statement
```

Comment:
This was the only "statement" interstitial before the end, I don't think it adds much.

Reply:
Agreed!